packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -14,6 +14,7 @@ import ReachabilityRequest from './request';
 const DEFAULT_TIMEOUT = 3000;
 const VIDEO_MESH_TIMEOUT = 1000;
 
+type IceCandidateResult = {clusterId: string; elapsed?: string; clientMediaIP?: string};
```

Comment:
I think I would prefer a name like "publicIP" or "publicMediaIp" so it's more clear to people reading the code that don't know much about ICE, webrtc etc

Reply:
renamed to `publicIP`